test-zone-data/MethodsV2/README.md

```diff
@@ -0,0 +1,1286 @@
+# Implementation of test scenarios for MethodsV2
+
+
+## Table of contents
+
+* [Background](#background)
+* [Root hint for scenarios](#root-hint-for-scenarios)
+* [Undelegated data for scenarios](#undelegated-data-for-scenarios)
+* [Data type in method output](#data-type-in-method-output)
+* [Scenarios](#scenarios)
+  * [GOOD-1](#good-1)
+  * [GOOD-2](#good-2)
+  * [GOOD-3](#good-3)
+  * [GOOD-UNDEL-1](#good-undel-1)
+  * [GOOD-UNDEL-2](#good-undel-2)
+
+## Background
+
+See [Specification of test scenarios for MethodsV2] for the specification of
+data and the definition of the scenarios.
+
+## Root hint for scenarios
+
+For all scenarios the root hint should be
```

Comment:
Just a minor cosmetic detail, but it feels more correct to name the root servers something like `ns1.root-servers.xa` instead of a TLD.